v2/internal/github/github.go

```diff
@@ -3,12 +3,51 @@ package github
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/charmbracelet/glamour"
 	"io"
 	"net/http"
+	"runtime"
 	"sort"
 	"strings"
 )
 
+func GetReleaseNotes(tagVersion string) string {
+	resp, err := http.Get("https://api.github.com/repos/wailsapp/wails/releases/tags/" + tagVersion)
```

Comment:
Should we `url.PathEscape` the `tagVersion`, just to be sure?

Reply:
Hell yes 😜